### DISCUSSION TITLE
pimcore_video not rendering in preview

### DISCUSSION THREAD

**Discussion by DavidSteinhaus** [pimcore/pimcore#13516] [Q&A / Support] [answered]
Hello , so I'm really new to pimcore and twig templates , I was trying to create a video editable that only works with youtube videos.
somehow it does work in edit mode but not in preview , can I get some help please.
here is my code: 
```
<div>
    <section id="youtube_video">
        {{ pimcore_video('youtubeVideo', {
                allowedTypes:['youtube'],
                thumbnail: 'youtube',
                width: 700,
                height: 400,
            }) 
        }}
    </section>
</div>
```

Like I said before it does work in edit mode I can add a youtube video Id and it works but when I go to preview there is nothing showing there.

I tried adding a get data method like this:
```
<div>
    <pre>
        {{ dump(pimcore_video('youtubeVideo').getData()) }}
    </pre>
</div>
```
and I do get some data.
if anyone have an idea on how to solve the problem I'd appreciate the help!

**Comment by dvesh3** [comment]
Seems related https://github.com/pimcore/pimcore/pull/13511 @kingjia90 Is the one you fixed related to the preview mode?

**Reply by kingjia90** [reply]
It's not related, the #13511, the code is fine in the case of `getYoutubeCode()`
https://github.com/pimcore/pimcore/blob/1f5e6be5d32eec3c611cd37807d2c6f1396d8f55/models/Document/Editable/Video.php#L721

**Comment by kingjia90** [comment]
I've tried the template and it works fine for me.

Could you please share your pimcore version, the output of the dump getData(), js console error and inspect the html code in the place where the video should appear to get an idea of what might be happening?

**Comment by DavidSteinhaus** [comment, marked as answer]
as @kingjia90 said the code works fine, I discovered that the Iframe generated by pimcore_video is being blocked by a cookie extension. disabling it or accepting cookings solved the problem. thanks a lot everyone!
